packages/markup-frontstage/src/components/ui/MarkupFrontstageProvider.tsx

```diff
@@ -220,7 +224,7 @@ export class MarkupFrontstageProvider extends FrontstageProvider {
         throw new UiError(
           MarkupFrontstage.loggerCategory(MarkupFrontstageProvider),
           `MarkupData.svg is invalid: ${
-          dom.getElementsByTagName("parsererror")[0].textContent
+            dom.getElementsByTagName("parsererror")[0].textContent
```

Comment:
should we check if this element exists and has length before accessing textContent?

Reply:
element exist check added please!